Update Marketo doc to include upsert leads toggle (#490)
Amplitude-Developer-Docs/amplitude-dev-center: docs/data/destinations/marketo-cohort.md

## Considerations

- Email values are case-sensitive. If you use email address to map users between Amplitude and Marketo, the email address must be consistent between both systems.

> - If a user on Amplitude does not already exist in Marketo, the option to create the user can be toggled on or off with the "Upsert Leads" toggle on the cohort sync screen.

Reviewer: ```suggestion
- Enable **Upsert Leads** on the cohort sync screen to create Amplitude users in Marketo if they don't already exist.
```

Reviewer: @ampcynthli -- can you review my suggestion to make sure it works for you?

Author: It looks great! Thank you
